shared/profile/prove-enter-username-container.js

```diff
@@ -35,7 +35,7 @@ export default connector.connect(
     }
 
     return {
-      canContinue: profile.usernameValid,
+      canContinue: true,
```

Comment:
The bug was validation occurs when continue selected but continue couldn't be selected since it was not enabled.